npm/npm_import.bzl

```diff
@@ -174,6 +175,7 @@ def npm_import(
             the `direct` attribute of the generated npm_link_package.
         run_lifecycle_hooks: If true, runs `preinstall`, `install` and `postinstall` lifecycle hooks declared in this
             package.
+        lifecycle_hooks_env: Environment variables applied to the lifecycle hooks scripts
```

Comment:
I think this needs a longer description of how the keys/values work

Reply:
Got it, I have updated the doc here, thanks.